public/locales/en/countries.json

```diff
@@ -6,27 +6,27 @@
     },
     "alerts": {
         "new-row": {
-            "error": "Error occurred trying to create a country.",
+            "error": "Error occured trying to create a country.",

```

Comment:
What's the reason to revert this typo fix?

Reply:
Thanks for the clarification @Marina-yoya

Reply:
We updated the branch wiht Marina and now this is not showing as revert